### PR TITLE
Email dive buddies

### PR DESCRIPTION
**Pull request by CatanaH**
added backend logic for sending email to dive buddies.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -809,10 +841,6 @@ def add_review():
     review.images.append(image)
 
   db.session.add(review)
-  
-  spot = Spot.query.filter_by(id=beach_id).first()
```
Comment (mjmayank1) [inline app/__init__.py:813]: did you mean to delete this?

File: app/__init__.py [hunk 2]
```diff
@@ -790,6 +791,37 @@ def add_review():
...
+    else:  
+      text += (f" I {activity} with {len(buddies)} other buddies.")
+    
+    buddy_message = Mail(
+      from_email=('hello@zentacle.com', 'Zentacle'),
+      to_emails=buddies)
+
+    buddy_message.template_id = 'd-8869844be6034dd09f0d7cc27e27aa8c'
+    buddy_message.dynamic_template_data = {
+      'display_name': user.username ,
```
Comment (mjmayank1) [inline app/__init__.py:814]: this should be user.display_name

File: app/__init__.py (hunk 1 above)
Comment (mjmayank1) [reply app/__init__.py:813]: nvm, seems like you moved it higher in the file

Comment (CatanaH) [reply app/__init__.py:813]: Yea I just moved it up so I could feed it into the email variables.

File: app/__init__.py (hunk 2 above)
Comment (CatanaH) [reply app/__init__.py:814]: ah, that would explain why it wasn't reading correctly.

File: app/__init__.py [hunk 3]
```diff
@@ -790,6 +791,37 @@ def add_review():
...
+      'spot_name': spot.name,
+      'spot_url': 'https://www.zentacle.com'+spot.get_url(),
+    }
+    if not os.environ.get('FLASK_ENV') == 'development':
```
[the remark this answers is not in the document]
Comment (CatanaH) [reply app/__init__.py:818]: ok, removed.

File: app/__init__.py [hunk 4]
```diff
@@ -780,6 +780,7 @@ def add_review():
   rating = request.json.get('rating')
   activity_type = request.json.get('activity_type')
   images = request.json.get('images') or []
+  buddies = request.json.get('buddy_array')
```
[the remark this answers is not in the document]
Comment (CatanaH) [reply app/__init__.py:783]: Added that in, for extra error handling

**Pull request merged**